server/aesdsocket.c

```diff
@@ -615,36 +600,89 @@ void* client_data_handler(void *thread_param)
             return NULL;
         }
 
-        result = write(dataFileDescriptor, receive_buffer, bytes_received);
-        if (result == ERROR)
+        // Check if the received string starts with "AESDCHAR_IOCSEEKTO:"
+        ioctl_check = strncmp(receive_buffer, ioctl_str, strlen(ioctl_str));
+        
+        if (ioctl_check == 0)
         {
-            syslog(LOG_ERR, "Unsuccessful file write operation");
-            return NULL;
-        }
+            struct aesd_seekto aesd_seekto_data;
+            sscanf(receive_buffer, "AESDCHAR_IOCSEEKTO:%d,%d", &aesd_seekto_data.write_cmd, &aesd_seekto_data.write_cmd_offset); 
```

Comment:
@Suraj-Ajjampur what happens if `sscanf` fails?

Reply:
This can become an issue, fixed with the latest commit.